op-challenger/scripts/alphabet/init_game.sh

```diff
@@ -1,43 +1,32 @@
 #!/bin/bash
-set -euo pipefail
 
-DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
-cd "$DIR"
+set -euo pipefail
 
-cd ../../
+SOURCE_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+CHALLENGER_DIR=$(echo ${SOURCE_DIR%/*/*})
+MONOREPO_DIR=$(echo ${SOURCE_DIR%/*/*/*})
 
+cd $MONOREPO_DIR
 make
```

Comment:
This no longer builds a fresh instance of the challenger every time that you are running the game because this `make` is being ran from the top-level makefile, not from the `op-challenger` makefile.

Reply:
The top-level `make` does re-build the challenger binary as well.

Reply:
Building the entire monorepo feels like overkill though unless we really need to. Don't want to spend much time on it but if we can trim this down to just building the things we actually need that would be a win.